detectable/src/test/java/com/synopsys/integration/detectable/detectables/ivy/IvyParseDetectableTest.java

```diff
@@ -0,0 +1,60 @@
+package com.synopsys.integration.detectable.detectables.ivy;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.Assertions;
+
+import com.synopsys.integration.bdio.model.Forge;
+import com.synopsys.integration.bdio.model.externalid.ExternalId;
+import com.synopsys.integration.bdio.model.externalid.ExternalIdFactory;
+import com.synopsys.integration.detectable.Detectable;
+import com.synopsys.integration.detectable.DetectableEnvironment;
+import com.synopsys.integration.detectable.extraction.Extraction;
+import com.synopsys.integration.detectable.functional.DetectableFunctionalTest;
+import com.synopsys.integration.detectable.util.graph.NameVersionGraphAssert;
+
+public class IvyParseDetectableTest extends DetectableFunctionalTest {
+    public IvyParseDetectableTest() throws IOException {
+        super("ivyparse");
+    }
+
+    @Override
+    protected void setup() throws IOException {
+        addFile(Paths.get("ivy.xml"),
```

Comment:
Should the optional file be included here as well?

Reply:
Good point, should test the name parsing.